spec/controllers/items_controller_spec.rb

```diff
@@ -10,5 +10,16 @@
     end
   end
 
-  # it { should route(:get, '/items/1').to(action: :show, id: 1) }
+  describe "GET #show/:id" do
+    it "returns http success" do
+      item = FactoryBot.create(:item)
```

Comment:
Could be `item = FactoryBot.create(:item)` or `let(:item) { create(:item }`

Reply:
done